pkg/virt-api/webhooks/validating-webhook/admitters/vmpool-admitter.go

```diff
@@ -41,7 +41,8 @@ import (
 )
 
 type VMPoolAdmitter struct {
-	ClusterConfig *virtconfig.ClusterConfig
+	ClusterConfig           *virtconfig.ClusterConfig
+	KubeVirtServiceAccounts map[string]struct{}
```

Comment:
Can you please unexpose this field? Same goes for all the similar fields

Reply:
Unfortunately, just a few admitters have a constructor function.
Even more unfortunate is the fact that `VMsAdmitter` has a constructor function, but is also initialized without it.
I will create a dedicated PR to fix it.
Having it here will increase the scope of this PR.